.github/workflows/Publish_NIMS.yml

```diff
@@ -38,6 +38,7 @@ jobs:
 
       - name: Check for lock changes
         run: poetry lock --check
+        working-directory: ./packages/service
```

Comment:
Apparently I can't comment on a potentially unrelated line, it looks like indentation on new 51 is wrong and won't apply the working dir